src/pages/en/guides/rss.md

```diff
@@ -114,6 +114,33 @@ export const get = () => rss({
 });
 ```
 
+### Including full post content
+
+By default, the Astro RSS integration does not support including the content of each of your posts in the feed itself. 
+
+However, if you create the list of RSS feed objects yourself, you can pass the content of Markdown files (not MDX), to the `content` key using the [`compiledContent` property](/en/guides/markdown-content/#exported-properties). We suggest using a package like [`sanitize-html`](https://www.npmjs.com/package/sanitize-html) in order to make sure that your content is properly sanitized, escaped, and encoded for use in the XML feed.
```

Comment:
Totally a nit but @sarah11918 should we be using `()` here to help indicate that’s a function you call rather than a static property?

```suggestion
However, if you create the list of RSS feed objects yourself, you can pass the content of Markdown files (not MDX), to the `content` key using the [`compiledContent()` property](/en/guides/markdown-content/#exported-properties). We suggest using a package like [`sanitize-html`](https://www.npmjs.com/package/sanitize-html) in order to make sure that your content is properly sanitized, escaped, and encoded for use in the XML feed.
```

Otherwise, looks great! Thanks for this contribution @smithbm2316 🙌